week2/homework/src/index.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+let fs = require('fs');
+let args = process.argv.slice(2);
+// console.log(args);
+
+let command = args[0];
+let todoItem = args[1];
+
```

Comment:
`fs`, `args`, `command` and `todoItem` are not re-assigned, so they have to be declared with `const`